### ISSUE TITLE
Ergonomics of a declaration with a null value

### ISSUE THREAD

**Issue by ThomasPO** [InseeFr/Lunatic-DSFR#174]
I need to display a statement dynamically : 
- if the answer to the previous question is "yes", then I display a content
- otherwise, I display nothing.

This works fine when I need to display the content. However, when the content is `null` then I get a visual with the statement's ergonomics (see screenshot below).
This would be fine if the ergonomics were not visible when the content is `null`.

![image](https://github.com/InseeFr/Lunatic-DSFR/assets/13235323/c7bf33d1-a0ba-4269-a342-59546885f01f)

**Comment by ddecrulle** [comment]
Hello,

Thank you for the report.

I would like to bring to your attention that the following JSON snippet:

```json
{
  "declarations": [
    {
      "label": {
        "type": "VTL|MD",
        "value": '""',
      },
      "position": "BEFORE_QUESTION_TEXT",
    },
  ]
}
```

will cause the same issue due to the interpretation of the label as Markdown.

The examples below will display nothing:

```json
{
  "declarations": [
    {
      "label": {
        "type": "VTL",
        "value": '""',
      },
      "position": "BEFORE_QUESTION_TEXT",
    },
  ]
}
```

```json
{
  "declarations": [
    {
      "label": {
        "type": "VTL", // And "VTL|MD"
        "value": 'false',
      },
      "position": "BEFORE_QUESTION_TEXT",
    },
  ]
}
```

```json
{
  "declarations": [
    {
      "label": {
        "type": "VTL", // and "VTL|MD"
        "value": 'null',
      },
      "position": "BEFORE_QUESTION_TEXT",
    },
  ]
}
```

```json
{
  "declarations": [
    {
      "label": {
        "type": "TXT",
        "value": '',
      },
      "position": "BEFORE_QUESTION_TEXT",
    },
  ]
}
```

**Comment by romaintailhurat** [comment]
Then we have a problem with VTL expression of the form `if condition then "string" else null`. They would have solved the problem if they had handled different return types (`string` and `null`) but they don't and [always return a type that is not null](https://github.com/InseeFr/Trevas-JS/blob/master/packages/trevas/src/visitors/Conditional.js#L46).

I think this behavior is consistent with the VTL spec, so our use case must be handle at the Lunatic DSFR level.

**Comment by ddecrulle** [comment]
Fixed in lunatic 3.0.5 which is included in Lunatic DSFR since 2.0.8
